src/components/ParameterRow.js

```diff
@@ -322,6 +322,11 @@ export default class ParameterRow extends Component {
             ) : null
           }
 
+          {/* Kong change -  Display error message in case of errors */}
+          { bodyParam ? null
+            : <ParameterRowError errors={this.getParameterRowErrorProps(paramWithMeta.get("errors"))} param={paramWithMeta.toJS()} />
+          }
```

Comment:
Why we are changing the position of that error? I think its not natural to display an error above place where it occur

Reply:
I think this would be for tab order logic. i.e. once you have an error appear, would you not want to have the next `tab` press take you to the field in order to resolve the error?

Reply:
I'll revert this for now.